fixtureeditor/editmode.cpp

```diff
@@ -31,6 +31,7 @@
 #include <QDebug>
 #include <QSize>
 #include <QAction>
+#include <QtGlobal>
```

Comment:
Please include the proper header(s)
When you include QtGobal, it means you have no clue of what is needed in this class

Reply:
It was because I used QOverload but build on the server failed, so I must change it...